scripts/translation/qaxml-pi.php

```diff
@@ -0,0 +1,71 @@
+<?php /*
++----------------------------------------------------------------------+
+| Copyright (c) 1997-2025 The PHP Group                                |
++----------------------------------------------------------------------+
+| This source file is subject to version 3.01 of the PHP license,      |
+| that is bundled with this package in the file LICENSE, and is        |
+| available through the world-wide-web at the following url:           |
+| https://www.php.net/license/3_01.txt.                                |
+| If you did not receive a copy of the PHP license and are unable to   |
+| obtain it through the world-wide-web, please send a note to          |
+| license@php.net, so we can mail you a copy immediately.              |
++----------------------------------------------------------------------+
+| Authors:     André L F S Bacci <ae php.net>                          |
++----------------------------------------------------------------------+
+
+# Description
+
+Compare processing instructions usage between two XML files.          */
+
+require_once __DIR__ . '/libqa/all.php';
+
+$argv   = new ArgvParser( $argv );
+$ignore = new OutputIgnore( $argv ); // may exit.
+$argv->complete();
+
+$list   = SyncFileList::load();
+
+foreach ( $list as $file )
+{
+    $source = $file->sourceDir . '/' . $file->file;
+    $target = $file->targetDir . '/' . $file->file;
+    $output = new OutputBuffer( "# qaxml.p" , $target , $ignore );
+
+    [ $s , $_ , $_ ] = XmlFrag::loadXmlFragmentFile( $source );
```

Comment:
Why are you reusing the variable `$_` twice? It's just going to overwrite it. And I don't think you are actually using this variable anywhere so why assign it at all?

Reply:
The function `loadXmlFragmentFile()` has 3 returns, but here the code is only interested on first return, so only one is "named" and all other cases are "discarded". 

Replacing `$_` by `_` causes `PHP Fatal error:  Assignments can only happen to writable values`.

And replacing `[ $s , $_ , $_ ]` by ` $s` causes `PHP Fatal error:  Uncaught TypeError: XmlFrag::listNodes(): Argument #1 ($node) must be of type DOMNode, array given`.

So I choose a repetitive and uninteresting name, that causes overwrite of variables that need to be positioned, but are unused.

Reply:
Yeah, but you don't have to assign it at all. `[$]` should do fine.

Reply:
Changed.